src/views/ajax.js

```diff
@@ -5,6 +5,36 @@ const arrowIcon = require('./open').arrowIcon;
 const icons = require('../assets/icons').default;
 const statusIcon = require('../assets/icons').statusIcon;
 
+/**
+ * removeOrigin - function to remove origin from a URL.
+ *
+ * @param {String} url URL to remove the origin from.
+ * @param {String} origin Origin to remove from URL.
+ */
+const removeOrigin = (url = '', origin = window.location.origin) => {
+  if (url.substr(0, origin.length) === origin) {
+      url = url.substr(origin.length, url.length);
+  }
+
+  return url;
+};
+
+/**
+ * removeOriginFromUrl - function to remove origin from a URL
+ *                       reguarding `http` and `https`.
+ *
+ * @param {String} url URL to remove the origin from.
+ * @param {String} origin Origin to remove from URL.
+ */
+const removeOriginFromUrl = (url = '', origin = window.location.origin) => {
+  url = url.trim();
+
+  url = removeOrigin(url, origin.replace(/^http\:\/\//, 'https://'));
```

Comment:
It does not and I think we have a bug on the Client? Because we do process the URLs with `http://` protocol but leave the URLs with `https://` protocol unattended which can be confusing since this is basically the same route?

![image](https://user-images.githubusercontent.com/1478800/28031270-59af253a-656c-11e7-9818-fac219e4e3c5.png)